app/code/Magento/Contact/Controller/Index/Post.php

```diff
@@ -70,8 +79,10 @@ public function execute()
             $this->messageManager->addErrorMessage($e->getMessage());
             $this->getDataPersistor()->set('contact_us', $this->getRequest()->getParams());
         } catch (\Exception $e) {
+            $this->logger->addError('An error occurred in Controller\\Index\\Post: ' . $e->getMessage());
```

Comment:
a) I'm not sure what M2 logging best practices are so I was throwing it in the two places I would expect to see it.
b) Should have used __CLASS__ here. 

I removed this line since the critical log could be enough? If you have any pointers to docs on best practices for core logging, I would appreciate it. 

Thanks!